packages/gatsby-theme-carbon/src/components/ResourceCard/ResourceCard.js

```diff
@@ -90,7 +90,15 @@ export default class ResourceCard extends React.Component {
       );
     } else {
       cardContainer = (
```

Comment:
```suggestion
	  // move the href logic up here and comment what it's doing
      cardContainer = (
```

Reply:
Okay, I think it's a bit easier to read/understand now.

Reply:
Agreed! Thank you.